utils.py

```diff
@@ -89,7 +91,7 @@ def multipart_encode(raw_crash, boundary=None):
 
     """
     if boundary is None:
-        boundary = uuid.uuid4().hex
+        boundary = '0a6e8e03c28e4e37abb57837f9ebf405'
```

Comment:
I switched this to a hard-coded boundary otherwise if we change the boundary, then the compressed size changes and then it fails the `post_crash` size check.